src/content/guides/language/sound-problems.md

```diff
@@ -494,6 +494,34 @@ var maximumOrNull =
 
 <hr>
 
+### Conflicting Superinterfaces
+
+A class which `implements` more than one superinterface must be able to
+implement valid overrides so each member requires compatible signatures across
+the superinterfaces.
+
+Superinterfaces must not include conflicting generics.
+A class cannot implement both `C<A>` and `C<B>`, including indirect
+superinterfaces.
+
+#### Example
+
+In the following code,
+class `C` will have conflicting generic interfaces and it will be impossible to
+define valide overrides for some members.
+
+<?code-excerpt "TODO"?>
```

Comment:
@atsansone - is there a doc describing how to set these up? Do we have any best practices for organization, or naming, or anything else I should be aware of?

Reply:
Yup. Kinda hidden: https://github.com/dart-lang/site-shared/blob/main/doc/code-excerpts.md